brave-unbreak.txt

```diff
@@ -172,6 +172,15 @@
 @@||wallpapershome.com/scripts/ads.js$script
 ! Anti-adblock: dreamdth.com
 @@||dreamdth.com/js/wutime_adblock/ads.js$script
+! spiegel.de (https://github.com/brave/brave-browser/issues/4201)
+||imagesrv.adition.com^$image,domain=spiegel.de
+||googletagservices.com/tag/js/gpt.js$script,domain=spiegel.de
+||mxcdn.net^$script,domain=spiegel.de
+||research.de.com^$script,domain=spiegel.de
+@@||googletagservices.com/tag/js/gpt.js$script,domain=spiegel.de
+@@||mxcdn.net^$script,domain=spiegel.de
+@@||research.de.com^$script,domain=spiegel.de
```

Comment:
This still seems over broad.  There is no reason to do the block / except pattern except when
1) we want the resource to be loaded
2) its being blocked by tracking protection (i.e. the domain appears in this list: https://github.com/brave/tracking-protection/blob/master/data/disconnect.json)